### ISSUE TITLE
The tags of attached PostgreSQL Flexible server and PostgreSQL Single server are both "PostgreSQL"

### ISSUE THREAD

**Issue by v-mengwe** [microsoft/vscode-cosmosdb#1904]
**OS:** All
**Build Version:** 20210720.10

**Repro steps:**

1. Attach the database account of the PostgreSQL Flexible server.
2. Attach the database account of the PostgreSQL Single server.
3. Check the tags of attached PostgreSQL Flexible server and PostgreSQL Single server.

**Expect:**
The tag of PostgreSQL Single server is "PostgreSQL Single" and the tag of PostgreSQL Flexible server is "PostgreSQL Flexible".
![image](https://user-images.githubusercontent.com/62053665/126478273-9113040d-91e0-452e-b5a2-e6c962dd0522.png)

**Actual:**
The tags of attached PostgreSQL Flexible server and PostgreSQL Single server are incorrect, and their tags are both "PostgreSQL".

**Comment by neelip** [comment]
The assumption is there is no obvious use case for users to attach an Azure Postgres server/database. As Flexible Servers are specific to Azure there is no support for them in "Attached" scenario.

@fiveisprime What do you think? Should we consider this use case as well?

**Comment by ejizba** [comment]
Users will attach with a connection string when they don't have full access to the subscription. I'm okay with the label just saying "PostgreSQL", but does this have any functional effects?

**Comment by neelip** [comment]
Even with latest release, users need to append 'sslmode=require' to their connection strings that they copied from the extension in order to attach azure accounts. But once the mentioned property is appended, all azure accounts are successfully attach, both Flexible and Single.

**Comment by ejizba** [comment]
But after they're attached, does it work when listing dbs, executing queries, etc.? If we're defaulting to single, I would expect this line might cause problems: https://github.com/microsoft/vscode-cosmosdb/blob/main/src/postgres/getClientConfig.ts#L27

That one might be an easy fix if we just do an array of all three certs in both cases. However, I'm not sure if there's other cases where the severType matters.

**Comment by neelip** [comment]
I just realized that the attach accounts never go through this... https://github.com/microsoft/vscode-cosmosdb/blob/main/src/postgres/getClientConfig.ts#L27
That's only for Azure accounts created through the extension, for attached accounts we just use the connection string itself.

**Comment by neelip** [comment]
I have ran some tests on PGAdmin using both Flexible and Single instances, both run successfully with mostly the same properties. 
The only difference being: 
- username (single: includes servername suffix and flexible doesn't)
- root certificate usage

Given that anything within a database is accessed using these values which are also used in PGAdmin...
` clientConfig = { user: username, password: password, ssl: true, host, port, database: databaseName };` 
if there is any problem it would be with accessing PostgresServerTreeItem or anything upstream. 

There shouldn't be any problem from what I have noticed with the Attached Accounts scenario given attached accounts are accessed through connectionStrings provided when attached.
